it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/trustControllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -157,77 +159,155 @@ class CheckYourAnswersControllerISpec extends AuditEnabledSpecHelper
   }
 
   "POST /identify-your-trust/check-your-answers-business" when {
-    "the EnableFullTrustJourney is enabled" should {
-      "redirect to the provided continueUrl" in {
+    "the EnableFullTrustJourney is enabled and identifier match is SuccessfulMatch (for example all postcodes are the same)" should {
+      "redirect to the provided continueUrl after contacting TrustKnownFacts api" in {
         enable(EnableFullTrustJourney)
+
         await(insertJourneyConfig(
           journeyId = testJourneyId,
           internalId = testInternalId,
           testTrustsJourneyConfig(businessVerificationCheck = true)
         ))
 
         stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+
         stubRetrieveUtr(testJourneyId)(OK, testUtrJson)
-        stubRetrieveCHRN(testJourneyId)(OK, testCHRN)
+        stubRetrieveSaPostcode(testJourneyId)(OK, testSaPostcode)
+        stubRetrieveTrustKnownFacts(testUtr)(OK, testKnownFactsJson(correspondencePostcode = testSaPostcode, declarationPostcode = testSaPostcode))
+        stubStoreStoreTrustsKnownFacts(testJourneyId)(OK, testTrustKnownFactsJson(correspondencePostcode = testSaPostcode, declarationPostcode = testSaPostcode))
```

Comment:
the store method shouldn't return the known facts. It should just return an OK status if stored successfully. A think a few of the store methods in the storage stub may need changing

Reply:
now it is clear ... I will fix it.